msg/Result.msg

```diff
@@ -0,0 +1,18 @@
+# Result code and message for service calls.
```

Comment:
Is this used in any messages?

Reply:
Yes, in all service responses.

Reply:
Sorry, I misread this as "Result code **for** message **and** service calls."  

Carry on.